### PR TITLE
attributes - React Basics

### PR DESCRIPTION
**Pull request by Freddie2024**

### CODE REVIEW

File: react-basics/react-basics_attributes/src/App.jsx [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
-  return <div>Replace me with an article</div>;
+  return <Article/>;
+}
+
+export function Article () {
+  return <article className="article">
+  <h2 className="article__title">Text of my Choice</h2>
+  <label htmlFor="input__field">Some input</label>
+  <input id="input__field"></input>
+  <a className="article__link" href="https://en.wikipedia.org/wiki/React_(JavaScript_library)"></a>
```
Comment (mgczarnievicz) [inline react-basics/react-basics_attributes/src/App.jsx]: Something is missing in the `<a>` tag. Were you able to see the link in the website?

**Comment by mgczarnievicz** [comment]
Hey, I don't know what you did,  but now you have 5000+ files that changes....

**Comment by mgczarnievicz** [comment]
The change, due to my comment is correct. But be carful when you push. 
Before doing anything regarding git, always do a `git status` to see that the file that has changes is the only one that you wanted to be. Then `git add` only the file you want.

File: react-basics/react-basics_attributes/src/App.jsx (hunk 1 above)
Comment (Freddie2024) [reply react-basics/react-basics_attributes/src/App.jsx]: I could see a dot that one could click on. But I added a text. And then something with node modules happened. I don't know how and why. Fatima helped me to fix it. And I don't know how to answer to your comments below.

**Pull request closed without merging**